pkg/common/configs/config.go

```diff
@@ -35,7 +36,7 @@ import (
 // set of scheduler resources.
 type SchedulerConfig struct {
 	Partitions []PartitionConfig
-	Checksum   [32]byte `yaml:"-" json:"-"`
+	Checksum   string `yaml:",omitempty" json:",omitempty"`
```

Comment:
I am wondering if we should split this in two. Checksum is always the current config checksum and "UpdateChecksum" is the checksum of the config we are updating. The reuse of the same field for two purposes is a bit strange and looks really weird in the code. What do you think?

Reply:
I agree that it is a bit strange, but if we would introduce a new `UpdateChecksum` that will be weird as well, because we will have to hack around the marshalling part. Also I can image that it may confuse the users as well, because while creating the input for the request it seems natural to extend it with `Checksum: .....`, how the getConfig is returning it, instead of changing it to `UpdateChecksum: ....`